kratos/includes/kratos_application.h

```diff
@@ -349,6 +349,8 @@ class KRATOS_API(KRATOS_CORE) KratosApplication {
     const Element mElement3D8N;
     const Element mElement3D10N;
 
+    const ConstitutiveLaw mConstitutiveLaw; //@technical-committee is this place ok?
```

Comment:
I would say it's ok

Reply:
At this moment it is ok, Nevertheless with Registry I would move each of them to their corresponding cpp file.